src/v3/src/components/LaunchAuthenticatorButton/LaunchAuthenticatorButton.tsx

```diff
@@ -30,10 +30,14 @@ const LaunchAuthenticatorButton: UISchemaElementComponent<{
     translations = [],
     options: {
       step,
+      deviceChallengeUrl,
     },
   } = uischema;
 
   const handleClick: ClickHandler = async () => {
+    if (deviceChallengeUrl) {
+      window.location.href = deviceChallengeUrl;
```

Comment:
We used `window.location.assign` in the `Redirect` component, so we should probably do the same here.